pbs_nodefile.c

```diff
@@ -0,0 +1,182 @@
+/*
+ * Plugin to set the PBS_NODEFILE env variable and its contents for the job
+ *
+ * (C) 2018 Ghent University
```

Comment:
this is not our standard license header block.

Reply:
Of course not. Code is based on other code, so I need to attribute.